src/js/common.js

```diff
@@ -1,12 +1,7 @@
-function trace(text) {
-  // This function is used for logging.
-  if (text[text.length - 1] === '\n') {
-    text = text.substring(0, text.length - 1);
-  }
-  if (window.performance) {
-    var now = (window.performance.now() / 1000).toFixed(3);
-    console.log(now + ': ' + text);
-  } else {
-    console.log(text);
-  }
+ /* exported trace */
+
+// Logging utility function.
+function trace(arg) {
+  var now = (window.performance.now() / 1000).toFixed(3);
+  console.log(now + ': ', arg);
```

Comment:
Will this print [object Object] for objects that don't have a stringifier?
This can be "caller's responsibility". It would be nice to have this implicit contract noted in the function comment.
I also note that we've dropped responsibility for removing trailing NL. That's good.